keyboards/skyloong/dt40/info.json

```diff
@@ -0,0 +1,69 @@
+{
+    "keyboard_name": "DT40",
+    "manufacturer": "skyloong",
+    "url": "https://github.com/JZ-Skyloong",
+    "maintainer": "LinLin",
+    "usb": {
+    	"vid": "0x1ea7",
+    	"pid": "0x0040",
+    	"device_version": "1.0.0"
+    },
+    "processor": "STM32F103",
+    "bootloader": "stm32-dfu",
```

Comment:
Pretty sure you're not using `stm32-dfu`.

Reply:
Ok, let me just make sure again.